packages/react/src/components/ContentSwitcher/ContentSwitcher.js

```diff
@@ -130,6 +130,7 @@ export default class ContentSwitcher extends React.Component {
   };
 
   render() {
+    const prefix = this.context;
```

Comment:
Wondering if you also need to update the classNames in this file to be `this.prefix` like you did for `ComposedModal` over in this PR: https://github.com/carbon-design-system/carbon/pull/9691/files. I think that may be why tests are failing?

Reply:
see: https://github.com/carbon-design-system/carbon/pull/9687#issuecomment-922062739

Reply:
setting `contextType` to `PrefixContext` should mean that `this.context` is our prefix for everything in that scope. In a few other files I then took the extra step of either assigning `prefix = this.context` for extra readability, but I'll probably revert that change in the other PRs

Reply:
looking into the failed tests now